.travis.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   include:
     - name: "Unit Tests"
       install:
-        - now install coverage linters pytest cylc rose
+        - now install coverage linters pytest isodatetime cylc rose
```

Comment:
Shouldn't have to install it here as the dependency is already in the `setup.py`.